pennylane/utils.py

```diff
@@ -189,7 +188,9 @@ def expand(U, wires, num_wires):
 
 @functools.lru_cache()
 def pauli_eigs(n):
-    """Returns the eigenvalues for :math:`A^{\otimes n}`, where :math:`A` is
+    r"""Pauli operator eigenvalues.
+
+    Returns the eigenvalues for :math:`A^{\otimes n}`, where :math:`A` is
```

Comment:
@smite it would be good to revert this change - the previous docstring structure was deliberate so that the full details would show up in the autosummary table. This results in simply showing 'Pauli operator eigenvalues', which is not entirely representative

Reply:
Ok, I'll fix it